tessellation/src/basic_shapes.rs

```diff
@@ -692,12 +666,9 @@ pub fn stroke_rounded_rectangle<Output: GeometryBuilder<StrokeVertex>>(
 
     let mut nums = radii.iter().map(|&radius| {
         if radius > 0.0 {
-
-            let arc_len = 0.5 * PI * radius;
+            let arc_len = 2.0 * PI * radius;
```

Comment:
Same here, `0.5 * PI * radius` should be correct because each corner is a quarter of a circle.